recipes/create_project.ipynb

```diff
@@ -140,7 +140,7 @@
     "# !pip install kili # uncomment if you don't have kili installed already\n",
     "from kili.client import Kili\n",
     "\n",
-    "api_key = os.getenv('KILI_USER_API_KEY')\n",
+    "api_key = os.getenv('KILI_API_KEY')\n",
     "api_endpoint = os.getenv('KILI_API_ENDPOINT') # If you use Kili SaaS, use the url 'https://cloud.kili-technology.com/api/label/v2/graphql'\n",
     "\n",
     "kili = Kili(api_key=api_key, api_endpoint=api_endpoint)"
```

Comment:
rm `api_key` argument

Reply:
I like this message!

Reply:
in authentification.py, in KiliAuth, I see this code:
```

if api_key is None:
            message = 'You need to provide an API KEY to connect.' \
                ' Visit https://docs.kili-technology.com/reference/creating-an-api-key'
            warnings.warn(message, UserWarning)
```
Do you get this message ?
If api is None, we could change that to an error of type AuthenticationFailed too but with a message more customized to the situation